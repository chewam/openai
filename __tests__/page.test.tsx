import { render } from "@testing-library/react"

import Page from "@/app/page"

jest.mock("react-textarea-autosize", () => {
  const ReactTextareaAutosize = () => <textarea>test</textarea>
  return ReactTextareaAutosize
})

it("renders homepage unchanged", () => {
  const { container } = render(<Page />)
  expect(container).toMatchSnapshot()
})
