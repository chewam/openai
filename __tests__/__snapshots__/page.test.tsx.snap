// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <main>
    <ul
      class="messages"
    >
      <li />
    </ul>
    <div
      class="form-container"
    >
      <form>
        <textarea>
          test
        </textarea>
        <button
          type="submit"
        >
          Submit
        </button>
      </form>
    </div>
  </main>
</div>
`;
